deploy/with-rbac.yaml

```diff
@@ -39,6 +39,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: STATSD_HOST
+              value: '127.0.0.1'
+            - name: STATSD_PORT
+              value: '8125'
```

Comment:
I'm surprised you needed to do these changes. Does test not use only test/manifest/...?